### PR TITLE
Update SHOW STATUS docs

### PR DESCRIPTION
**Pull request by enkilee**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Fix PR15363 https://github.com/pingcap/docs/pull/15363

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15363
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-status.md [hunk 1]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
 
 # SHOW [GLOBAL|SESSION] STATUS
 
-`SHOW [GLOBAL|SESSION] STATUS` 语句用于提供 MySQL 兼容性，对 TiDB 没有作用。因为 TiDB 使用 Prometheus 和 Grafana 而非 `SHOW STATUS` 来进行集中度量收集。
+`SHOW [GLOBAL|SESSION] STATUS` 语句用于提供 MySQL 兼容性，TiDB 使用 Prometheus 和 Grafana 进行集中度量收集, 而不是使用 `SHOW STATUS` 进行各种度量收集。
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-status.md:9]:
```suggestion
`SHOW [GLOBAL|SESSION] STATUS` 语句用于提供 MySQL 兼容性。对于大部分监控指标，TiDB 使用 Prometheus 和 Grafana 来集中收集，而不是使用 `SHOW STATUS`。
```

File: sql-statements/sql-statement-show-status.md [hunk 2]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
...
 
-## 语法图
-
-**ShowStmt:**
-
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
+详细介绍请参考 [服务器状态变量](/status-variables.md)
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-status.md:11]:
```suggestion
详细介绍请参考[服务器状态变量](/status-variables.md)。
```

File: status-variables.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+title: 服务器状态变量
+summary: 使用状态变量查看系统和会话状态
+---
+
+# 服务器状态变量
+
+服务器状态变量提供有关服务器全局状态和 TiDB 中当前会话状态的信息。大多数变量与 MySQL 兼容。
+
+你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态, 当前会话状态可以使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令。
```
Comment (ran-huang) [inline status-variables.md:10]:
```suggestion
你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态，使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令检索当前会话状态。 
```

File: status-variables.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+summary: 使用状态变量查看系统和会话状态
+---
+
+# 服务器状态变量
+
+服务器状态变量提供有关服务器全局状态和 TiDB 中当前会话状态的信息。大多数变量与 MySQL 兼容。
+
+你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态, 当前会话状态可以使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令。 
+
+此外, [FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) 命令与 MySQL 兼容。
```
Comment (ran-huang) [inline status-variables.md:12]:
```suggestion
此外，[FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) 命令与 MySQL 兼容。
```

File: status-variables.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+## 系统变量
```
Comment (ran-huang) [inline status-variables.md:14]:
```suggestion
## 变量参考
```

File: status-variables.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+### Compression
+
+- 作用域: SESSION
```
Comment (ran-huang) [inline status-variables.md:18]: Global comment: we should use *full-width* colon for Chinese text. Please update the rest of the document.
```suggestion
- 作用域：SESSION
```

Comment (enkilee) [reply status-variables.md]: Done

**Comment by enkilee** [comment]
@ran-huang already fix.

File: status-variables.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态，使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令检索当前会话状态。  
+
+此外，[FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) 命令与 MySQL 兼容。
+
+## 变量参考
+
+### Compression
+
+- 作用域：SESSION
+- 类型：字符串
```
Comment (ran-huang) [inline status-variables.md:19]:
```suggestion
- 类型：布尔值
```

File: status-variables.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+- MySQL 是否使用压缩协议。
+
+### Compression_algorithm
+
+- 作用域：SESSION
+- 类型：字符串
+- MySQL 压缩协议使用的算法。
```
Comment (ran-huang) [inline status-variables.md:26]:
```suggestion
- MySQL 协议使用的压缩算法。
```

File: status-variables.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+### Compression_level
+
+- 作用域：SESSION
+- 类型：整数型
+- MySQL 压缩协议使用的等级。
```
Comment (ran-huang) [inline status-variables.md:32]:
```suggestion
- MySQL 协议使用的压缩等级。
```

File: status-variables.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+### Ssl_cipher_list
+
+- 作用域：SESSION | GLOBAL
+- 类型：字符串
+- 服务器支持的 TLS 加密套件列表.
+
+### Ssl_server_not_after
+
+- 作用域：SESSION | GLOBAL
+- 类型：时间
```
Comment (ran-huang) [inline status-variables.md:49]:
```suggestion
- 类型：日期
```

File: status-variables.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+- 作用域：SESSION | GLOBAL
+- 类型：整数型
+- DDL schema 使用的版本。
+
+### last_plan_binding_update_time <span class="version-mark">New in v5.2.0</span>
+
+- 作用域：SESSION
+- 类型：时间戳
+- 上次计划更新的日期时间。
```
Comment (ran-huang) [inline status-variables.md:86]:
```suggestion
- 最后一次计划绑定更新的日期时间。
```

File: status-variables.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+### server_id
+
+- 作用域：SESSION | GLOBAL
+- 类型：字符串
+- 服务器的通用唯一识别码（UUID）。
```
Comment (ran-huang) [inline status-variables.md:92]:
```suggestion
- 服务器的通用唯一识别码 (UUID)。
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM. Thank you for the great translation!

File: status-variables.md (hunk 12 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

File: status-variables.md (hunk 11 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

File: status-variables.md (hunk 10 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

File: status-variables.md (hunk 9 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

File: status-variables.md (hunk 8 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

File: status-variables.md (hunk 7 above)
Comment (enkilee) [reply status-variables.md]: Done.thanks

**Comment by enkilee** [comment]
> Rest LGTM. Thank you for the great translation!

@ran-huang Done.Thanks

**Review by ran-huang — APPROVED** [review]

File: sql-statements/sql-statement-flush-status.md [hunk 13]
```diff
@@ -100,8 +100,9 @@ show status;
 
 ## MySQL 兼容性
 
-* `FLUSH STATUS` 语句仅用于提供 MySQL 兼容性。
+* `FLUSH STATUS` 语句与 MySQL 兼容性。
```
Comment (qiancai) [inline sql-statements/sql-statement-flush-status.md:103]:
```suggestion
* `FLUSH STATUS` 语句与 MySQL 兼容。
```

File: sql-statements/sql-statement-show-status.md [hunk 14]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
...
+Scope ::=
+    ( 'GLOBAL' | 'SESSION' )
+ShowLikeOrWhere ::=
+    "LIKE" SimpleExpr
+|   "WHERE" Expression
+```
 
 ## 示例
 
 {{< copyable "sql" >}}
```
Comment (qiancai) [inline sql-statements/sql-statement-show-status.md:28]:
```suggestion
```

File: sql-statements/sql-statement-show-status.md [hunk 15]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
...
+| Ssl_version                   |                                      |
+| Uptime                        | 1409                                 |
+| ddl_schema_version            | 116                                  |
+| last_plan_binding_update_time | 0000-00-00 00:00:00                  |
+| server_id                     | 61160e73-ab80-40ff-8f33-27d55d475fd1 |
++-------------------------------+--------------------------------------+
+13 rows in set (0.00 sec)
 ```
 
 {{< copyable "sql" >}}
```
Comment (qiancai) [inline sql-statements/sql-statement-show-status.md:55]:
```suggestion
```

File: sql-statements/sql-statement-show-status.md [hunk 16]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
...
+| ddl_schema_version    | 116                                  |
+| server_id             | 61160e73-ab80-40ff-8f33-27d55d475fd1 |
++-----------------------+--------------------------------------+
+9 rows in set (0.00 sec)
 ```
 
 ## MySQL 兼容性
 
-`SHOW [GLOBAL|SESSION] STATUS` 语句仅用于提供 MySQL 兼容性。
+`SHOW [GLOBAL|SESSION] STATUS` 语句与 MySQL 兼容性。
```
Comment (qiancai) [inline sql-statements/sql-statement-show-status.md:79]:
```suggestion
`SHOW [GLOBAL|SESSION] STATUS` 语句与 MySQL 兼容。
```

File: status-variables.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
+---
+title: 服务器状态变量
+summary: 使用状态变量查看系统和会话状态
+---
+
+# 服务器状态变量
+
+服务器状态变量提供有关服务器全局状态和 TiDB 中当前会话状态的信息。大多数变量与 MySQL 兼容。
+
+你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令检索全局状态，使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令检索当前会话状态。
```
Comment (qiancai) [inline status-variables.md:10]:
```suggestion
你可以使用 [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) 命令查看全局状态，使用 [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) 命令查看当前会话状态。  
```

File: sql-statements/sql-statement-show-status.md [hunk 18]
```diff
@@ -6,68 +6,79 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-status/','/docs-cn/dev
...
+`SHOW [GLOBAL|SESSION] STATUS` 语句用于提供 MySQL 兼容性。对于大部分监控指标，TiDB 使用 Prometheus 和 Grafana 来集中收集，而不是使用 `SHOW STATUS`。
 
-## 语法图
-
-**ShowStmt:**
-
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
+详细介绍请参考[服务器状态变量](/status-variables.md)。
```
Comment (qiancai) [inline sql-statements/sql-statement-show-status.md:11]:
```suggestion
该语句输出中各变量的详细介绍，请参考[服务器状态变量](/status-variables.md)。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**